airflow/www/jest.config.js

```diff
@@ -28,6 +28,7 @@ const config = {
   moduleNameMapper: { // Listing all aliases
     '^src/(.*)$': '<rootDir>/static/js/$1',
   },
+  transformIgnorePatterns: [], // transform node_modules
```

Comment:
@pierrejeambrun were you going to update this array to only ignore the specific files from react-markdown?

Reply:
Just updated the PR !